src/pyfmi/fmi_algorithm_drivers.py

```diff
@@ -100,10 +100,23 @@ class AssimuloFMIAlgOptions(OptionBase):
             solver.
             Default: "Default"
 
+        dynamic_diagnostics --
+            If True, will enable logging of diagnostics data and requires that
+            the option 'result_handling' is set to 'binary', otherwise an exception is raised.
+            If True, the debug file will not be generated.

```

Comment:
All users will not now what "the debug file" refers to. I suggest to remove this line, especially with the update above.